stl/inc/xcall_once.h

```diff
@@ -28,12 +28,14 @@ _EXPORT_STD struct once_flag { // opaque data structure for call_once()
     void* _Opaque;
 };
 
+#ifdef _CRTBLD

```

Comment:
Should we file a TODO to move anything `#ifdef _CRTBLD` from public to private headers?

Reply:
Filed #3941.